service-worker/kill-switch/index.html

```diff
@@ -0,0 +1,61 @@
+---
+feature_name: "Service Worker Sample: Kill Switch"
+chrome_version: 45
+feature_id: 6561526227927040
+---
+
+<h3>Background</h3>
+<p>
+  This samples demonstrates one approach to implementing a service worker
+  "<a href="https://en.wikipedia.org/wiki/Kill_switch">kill switch</a>". This pages registers a
+  service worker that has an undesirable <code>fetch</code> handler, which unconditionally
+  returns useless HTML in response to every fetch request. This simulates the "worst case scenario"
+  for a buggy or poorly implemented service worker. <strong>Once you reload this page, you'll see the
+  undesirable service worker in action.</strong>
+</p>
+
+<p>
+  Before getting into the technical details of the kill switch, it's worth pointing out that even
+  in this "worst case scenario", browsers offer built-in remediations that might prove sufficient.
+  One remediation is to
+  <a href="https://en.wikipedia.org/wiki/Wikipedia:Bypass_your_cache">shift-reload</a>, which will
+  bypass the service worker and serve the page directly from the network.
+</p>
+
+<p>
+  Another remediation is simply to update your service worker script to resolve the bug, and
+  redeploy it to your web server. As soon as the service worker script is expired from a
+  browser's HTTP cache, the updated service worker script will end up being registered. You
+  can use the <a href="../immediate-control/">immediate control</a> pattern in your updated script
+  to ensure that the newly registered service worker takes control over existing tabs. Even if
+  you've inadvertently set the HTTP cache control headers to expire at some point in the far future,
+  a cached copy of the service worker script older than
+  <a href="https://github.com/slightlyoff/ServiceWorker/issues/514">24 hours</a> is automatically
+  ignored, and a call to <code>navigator.serviceWorker.register()</code> will fetch a fresh copy
+  from the network at that point. So the worst-case scenario is that users might have to wait a day
+  until their previously cached service worker script expires.
+</p>
+
+<p>
+  That being said, overly cautious developers might want to add in "kill switch" functionality
```

Comment:
I might change the word 'overly' to something like 'particularly'. The current wording basically says that you *never* need to do this, and if you want to do it anyway then you are just being paranoid.